package/skelcd-control-MicroOS.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Mon Feb 22 21:45:19 EST 2021 - Neal Gompa <ngompa13@gmail.com>
+
+- Declare package manager patterns for each MicroOS role
```

Comment:
in general change looks good. We just need here some ID ( bsc#<num> for bugzilla entry or if it is feature then jsc#<id>), it is needed to match reason for change.
Thanks

Reply:
I don't have a specific Bugzilla or SUSE Jira issue though. I made this change to keep things in-line with what's happening in the pattern changes (otherwise things will be very broken).

Reply:
@jreidinger I made a bug and changed this accordingly.